src/m365/spo/commands/page/page-add.ts

```diff
@@ -334,7 +334,7 @@ class SpoPageAddCommand extends SpoCommand {
       },
       {
         option: '-l, --layoutType [layoutType]',
-        autocomplete: ['Article', 'Home']
+        autocomplete: ['Article', 'Home', 'SingleWebPartAppPage', 'RepostPage', 'HeaderlessSearchResults', 'Spaces', 'Topic']
```

Comment:
To avoid repeating supported values, let's centralize them and references across both `page add` and `page set` and autocomplete, validation and error messages